src/config.rs

```diff
@@ -31,14 +32,15 @@ impl Default for Config
             CLEANUP_TEMPORARY_FILES: None,
             CSRFTOKEN: Some("".to_owned()),
             DATABASE_URL: "./db/db.sqlite".to_owned(),
-            DEBUG: None, // no entry in default config, defaults to false
+            DEBUG: None,
             DONTDOWNLOADME_FILEPATH: Some("./config/dontdownloadme.txt".to_owned()),
             DOWNLOADME_FILEPATH: Some("./config/downloadme.txt".to_owned()),
             LIBRARY_PATH: "./hentai/".to_owned(),
             LIBRARY_SPLIT: None,
             NHENTAI_TAGS: None,
             SLEEP_INTERVAL: Some(50000),
+            TITLE_TYPE: Some("TITLE_ENGLISH".to_owned()), // default to English title for backward compatibility
```

Comment:
Sorry I have used Strings with python and alittle bit of C# but Rust is new grounds. Trying to figure things out. With some messing around I think I got the enums figured out
I also believe that using snake_case should be correct here so that it can be converted between caps or lowercase.